src/Layout/Presentation/DistrictOverview/VulnerableDistrictSeatText.tsx

```diff
@@ -1,21 +1,26 @@
 import * as React from "react";
 import { VulnerableDistrictSeat } from "../../../utilities/district";
+import { Dictionary } from "lodash";
 
 export interface VulnerableDistrictSeatTextProps {
     mostVulnerable?: VulnerableDistrictSeat | undefined;
+    partyMap: Dictionary<string>;
 }
 
 export class VulnerableDistrictSeatText extends React.Component<VulnerableDistrictSeatTextProps, {}> {
     render() {
         if (this.props.mostVulnerable) {
+            const partyMap = this.props.partyMap;
+            const winnerPartyCode = this.props.mostVulnerable.winner.partyCode;
+            const runnerUpPartyCode = this.props.mostVulnerable.runnerUp.partyCode;
             return (
                 <React.Fragment>
                     {" Det mest utsatte sistemandatet (relativt til kvotient) var i "}
                     {this.props.mostVulnerable.district}
                     {" og ble vunnet av "}
-                    {this.props.mostVulnerable.winner.partyCode}
+                    <b>{partyMap[winnerPartyCode]}</b>
```

Comment:
Isn't it better to use classes to define appearance / visual behaviour like this?